index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <nav class="toolbar">
+            <div class="logo">JOSEPH</div>
```

Comment:
- [ ] [OPTIONAL] Since this is the logo of your website, Please make it a link using the anchor tag. I think this should be a link that navigates to the homepage of the website.